pandas/tests/series/test_analytics.py

```diff
@@ -336,6 +336,24 @@ def test_describe(self):
                           index=['count', 'unique', 'top', 'freq'])
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize("tz, name", [
+        (None, "tz-naive"),
+        ("US/Eastern", "tz-aware")
+    ])
+    def test_describe_dt(self, tz, name):
+        start = Timestamp(2018, 1, 1)
+        end = Timestamp(2018, 1, 5)
+        s = Series(date_range(start, end, tz=tz), name=name)
```

Comment:
can you add a test for frame as well (in the appropraite file)

Reply:
done